task_02.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Creates a program for login and authentication"""
+
+
+import authentication
+import getpass
+
+
+def login(username, max_attempts=3):
+    """This function loops through user input to authenticate password
+
+    Args:
+        It takes 2 arguments, username and max_attempts, with 3 as default max.
+
+    Returns:
+        A boolean
+    """
+
+    authenticated = False
+    attempt = 0
+    while (attempt < max_attempts) or (not authenticated):
```

Comment:
This is your infinite loop -- because you used an `or` if _either_ condition returned `True` it would continue.